src/components/services-view/service-view.classes.ts

```diff
@@ -0,0 +1,181 @@
+import {allCategories} from '../../constants';
+import * as _ from 'lodash';
+
+interface IServiceItem {
+    icon: string;
+    name: string;
+    category: string;
+    subCategory: string;
+    description: string;
+    longDescription: string;
+    data: any;
+}
+
+export class CatalogServiceItem implements IServiceItem {
+    public icon: string;
+    public name: string;
+    public category: string;
+    public subCategory: string;
+    public description: string;
+    public longDescription: string;
+    public data: any;
+
+    constructor (catalogService: any) {
+        this.data = catalogService;
+        this.icon = this.getIcon();
+        this.name = this.getName();
+        this.description = this.getDescription();
+        this.longDescription = this.getLongDescription();
+        this.subCategory = this.getSubCategory();
+        this.category = this.getCategory();
```

Comment:
yeah they can have many tags, and you might have tags that put something into more then one category.  Our existing catalog supports this.  As an example, I might have a template that is an example with a database and a php app, i might tag that both with "database" and "php" so that it shows up in both places.  The idea is to catch people from multiple points of entry since they might look for things in different ways.